openslides_backend/action/actions/user/delete.py

```diff
@@ -9,17 +8,22 @@
 from ...util.default_schema import DefaultSchema
 from ...util.register import register_action
 from .check_temporary_mixin import CheckTemporaryNoForInstanceMixin
+from .user_scope_permission_check_mixin import UserScopePermissionCheckMixin
 
 
 @register_action("user.delete")
-class UserDelete(CheckTemporaryNoForInstanceMixin, DeleteAction):
+class UserDelete(
+    CheckTemporaryNoForInstanceMixin, UserScopePermissionCheckMixin, DeleteAction
```

Comment:
CheckTemporaryNoForInstanceMixin: This has been needed for temporary users

Reply:
No, but that work is part of #679 and I did not want to mix it in here